src/NServiceBus.Transport.Msmq/Scripts/NServiceBus.Transport.Msmq.targets

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This new file should be a props file, not a targets file.

Reply:
Check